app/Http/Controllers/mainController.php

```diff
@@ -8,125 +8,151 @@
         return view('user.settings', $data);
     }
     //Thommi - dsjfhdsjfhdsfjs
-    function myWatchs()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function myWatchs() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.mywatchs', $data);
     }
-
-    function myRatings()
-    {
-        $data = ['LoggedUserInfo' => User::where('id', '=', session('LoggedUser'))->first()];
-
+    function myRatings() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
         return view('user.myratings', $data);
     }
+    function myMovies() {
+        $data = ['LoggedUserInfo'=>User::where('id','=', session('LoggedUser'))->first()];
+        return view('user.mymovies', $data);
+    }
+    //WARNING!!
+    //Remember all functions added here are the ones inside web authCheck!
+    //SELECT * FROM genre_movie INNER JOIN movies ON genre_movie.genre_id = movies.id ORDER BY genre_movie.genre_id DESC
+    public function getInfo($id) {
+        $page = DB::select("SELECT  movies.id,
+                                    movies.title,
+                                    movies.director,
+                                    movies.description,
+                                    movies.writer,
+                                    movies.year,
+                                    movies.img,
+                                    group_concat(concat(' <a href=''/genre/' , genres.genre , ''' class=''hover:underline italic''>' , genres.genre , '</a>')) AS genre
+                                    FROM movies
+                                    INNER JOIN genre_movie ON genre_movie.movie_id = movies.id
+                                    INNER JOIN genres ON genre_movie.genre_id = genres.id WHERE movies.id = $id
+                                    GROUP BY movies.id, movies.title, movies.year");
+        // dd($page);
+        return view('movie', array('page' => $page));
+    }
+    public function getGenre($id) {
+        $genreList  =   DB::select('SELECT  movies.id,
+                                            movies.title,
+                                            movies.img,
+                                            group_concat(genres.genre) AS genre
+                                            FROM movies
+                                            INNER JOIN genre_movie ON genre_movie.movie_id = movies.id
+                                            INNER JOIN genres ON genre_movie.genre_id = genres.id WHERE genre_movie.genre_id='.$id.'
+                                            GROUP BY movies.id, movies.title, movies.img');
+        // dd($genreList);
+        return view('genre', array('genreList' => $genreList));
+    }
 
-    function myMovies()
+    function review() {
+        //
+        return view('review');
+    }
+
+    function store(Request $request)
```

Comment:
Should review have its own controller?

Reply:
Inte än, först ser jag till att den fungerar som en $id och sen lägger jag till en separat review. Prioritet är att den ska fungera som det ska och inte har några bugs.

Reply:
vi kollar fortarande med Mehrdad och Tatjana.